src/jdk.jlink/linux/classes/jdk/tools/jlink/internal/plugins/StripNativeDebugSymbolsPlugin.java

```diff
@@ -22,6 +22,13 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
+
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
+ * ===========================================================================
+ */
+
```

Comment:
My preference is that you not add blank lines before the `package` declaration as there were none present before this change. I wasn't going to mention that, but since this should be squashed anyway, please remove lines 25 and 31.

Reply:
I removed line `25` and `31` and then performed a `squash`.